packages/descendants/src/Highlight/Highlight.spec.ts

```diff
@@ -0,0 +1,17 @@
+describe('packages/descendants/highlight', () => {
+  describe('useHighlight', () => {
+    test.todo('returns the current highlight object, and setter function(s)');
+    test.todo('calling the setter updates the highlight object');
+    test.todo(
+      '`onInit` option is called once when descendants are initialized',
+    );
+    test.todo('`onChange` option is called when the highlight object changes');
+    test.todo(
+      '`filter` option prevents relevant descendants from being highlighted',
+    );
+  });
+
+  describe('useHighlightContext', () => {
+    test.todo('returns the current highlight object');
```

Comment:
is there some reason we're unable to add these at this time?

Reply:
...I just forgot...